src/test/models/users.model.js

```diff
@@ -6,29 +6,20 @@ import '../../server/models/users.model';
  * Test for mongodb create.
  */
 
-let should = require('should');
+const should = require('should');
 let user = require('mongoose').model('user');
+let u = {
+  name: 'yoda'
+};
 
 describe('Users: models', function () {
   describe('#create()', function () {
-    it('should create a new User', function (done) {
+    it('should create a new User and remove', function (done) {
       // Create a User object to pass to User.create()
-      var u = {
-        name: 'yoda'
-      };
       user.create(u, function (err, createdUser) {
         should.not.exist(err);
         createdUser.name.should.equal('yoda');
-        done();
-      });
-    });
-  
-    it('should delete new User', function (done) {
-      // Create a User object to pass to User.create()
-      var u = {
-        name: 'yoda'
-      };
-      user.find(u).remove(function() {
+        createdUser.remove();
```

Comment:
In all of these tests, create() should actually do a lookup from the db (rather than looking at the createdUser) so the object is actually inserted in the db. Once that verification is done, then we can do createUser.remove(). 

Same goes for line 24 of users.controller.test.js; we should just create a few users A and B, then pull up all users using getMentorList() method, and check if the returned list contains precisely A and B (and not the current session user), and finally remove A and B. 

Can you add these in the TODO comments? Without these, we're merely testing for user.create() method which isn't even our own code.